mllib/src/main/scala/org/apache/spark/mllib/tree/configuration/BoostingStrategy.scala

```diff
@@ -70,20 +70,31 @@ object BoostingStrategy {
 
   /**
    * Returns default configuration for the boosting algorithm
+   * @param algoStr Learning goal.  Supported:
+   *             [[org.apache.spark.mllib.tree.configuration.Algo.Classification]],
```

Comment:
It is clearer if we say `"Classification" and "Regression"` instead of list classes for `algo: String`.